zulipterminal/cli/run.py

```diff
@@ -285,6 +286,19 @@ def main(options: Optional[List[str]]=None) -> None:
         else:
             theme_to_use = zterm['theme']
 
+        ZULIPRC_CONFIG = 'in zuliprc file'
```

Comment:
I'd prefer we didn't rely on testing against this - this is intended for output purposes.

Perhaps we can move the validation into `parse_zuliprc`?

Reply:
But `parse_zuliprc` is specifically for parsing the zuliprc file, rather than validation. All other validations take place in the `main` function.